### ISSUE TITLE
Depth Video Preprocessing during training

### ISSUE THREAD

**Issue by maoqingsunny** [Tencent/DepthCrafter#45]
Thank you for providing this amazing work for depth video generation! I am currently trying to fine-tune the model on my dataset but have poor results. I suspect the issue might be how I preprocess the depth videos during training.
Could you provide some details about the depth video preprocessing pipeline used during training?  Thank you for your time and support!

**Comment by gaoxiangjun** [comment]
Could you refer to `benchmark/dataset_extract` for details?

**Comment by wbhu** [comment]
During training, the depth video is normalized to [0,1] in a sequence-wise manner. Please refer to Sec. 3.2 in the paper

**Comment by wbhu** [comment]
closed as no further questions
